critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
+  'use strict';
+
+  var clearClass = 'rating-clear',
```

Comment:
Don't be afraid to modify a library code if it can be improved or adjusted for the way we use it. Not every library published on the internet is perfect. :)